package.json

```diff
@@ -100,25 +100,24 @@
     "elasticsearch": "^14.2.1",
     "elasticsearch-browser": "^14.2.1",
     "encode-uri-query": "1.0.0",
-    "even-better": "7.0.2",
     "execa": "^0.10.0",
     "expiry-js": "0.1.7",
     "extract-text-webpack-plugin": "3.0.1",
     "file-loader": "1.1.4",
     "font-awesome": "4.4.0",
     "glob": "^7.1.2",
     "glob-all": "^3.1.0",
+    "good": "github:joshdover/good#b9adb7f4e90bca43ef6c01bfcbd9359481a93bbc",
```

Comment:
Will this be published before this gets merged?

Reply:
I have an email thread with the maintainer going. He is needing some convincing to accept our changes, so as of now, I'm not 100% sure. Happy to move this / publish to NPM as need to get this live.